tests/snuba/sessions/test_sessions.py

```diff
@@ -23,10 +52,7 @@ def make_24h_stats(ts):
     return _make_stats(datetime.utcfromtimestamp(ts).replace(tzinfo=pytz.utc), 3600, 24)
 
 
-class ReleaseHealthMetricsTestCase(SessionMetricsTestCase):
-    backend = MetricsReleaseHealthBackend()
-
-
+@parametrize_backend
 class SnubaSessionsTest(TestCase, SnubaTestCase):
     backend = SessionsReleaseHealthBackend()
```

Comment:
nit: Should we move this into `@parametrize_backend`?

Reply:
yup ok!